src/riak_kv_vnode.erl

```diff
@@ -579,7 +579,12 @@ handle_command({hashtree_pid, Node}, _, State=#state{hashtrees=HT}) ->
             %% after the vnode was already running
             case HT of
                 undefined ->
-                    State2 = maybe_create_hashtrees(State),
+                    case riak_repl2_fssource:get_strategy() == aae of
```

Comment:
@rsltrifork, we can not have riak_kv reference repl code, which is not open source and is not a dependency of riak_kv